extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -191,6 +199,16 @@ public function onEditPageLayoutExecute( \EditPageLayoutController $editPage ) {
 	}
 
 	/**
+	 * Determines whether a title is a draft in conversion mode initiated
+	 * @param IRequest $request
+	 * @param \Title $title
+	 * @return bool
+	 */
+	public function isTemplateDraftConversion( IRequest $request, \Title $title ) {
```

Comment:
I think that since we use it in `TemplateClassificationView` as well it should not be a part of the `Hooks` class.